### DISCUSSION TITLE
How to get the reference to the data grid?

### DISCUSSION THREAD

**Discussion by nanandn** [Comcast/react-data-grid#2550] [Q&A]
Hi, how do I set/get the ref to the data table node? I pass in a "ref" attribute, but the ref.current looks like

```
current:
scrollToColumn: ƒ scrollToColumn(idx)
scrollToRow: ƒ scrollToRow(rowIdx)
selectCell: ƒ selectCell(position, openEditor)
```

I am using react-data-grid": "7.0.0-canary.16

Thanks
Anand

**Comment by igoriok1994** [comment]
> #2038 Also related

**Comment by amanmahajan7** [comment]
you can use `gridRef.element`
